mara_db/shell.py

```diff
@@ -246,14 +250,25 @@ def copy_to_stdout_command(db: object,
 
 
 @copy_to_stdout_command.register(str)
-def __(alias: str, header: bool = None, footer: bool = None, delimiter_char: str = None, csv_format: bool = None):
+def __(alias: str, header: bool = None, footer: bool = None, delimiter_char: str = None, csv_format: bool = None,
+       pipe_format: Format = None):
     return copy_to_stdout_command(dbs.db(alias), header=header, footer=footer,
-                                  delimiter_char=delimiter_char, csv_format=csv_format)
+                                  delimiter_char=delimiter_char, csv_format=csv_format,
+                                  pipe_format=pipe_format)
 
 
 @copy_to_stdout_command.register(dbs.PostgreSQLDB)
 def __(db: dbs.PostgreSQLDB, header: bool = None, footer: bool = None,
-       delimiter_char: str = None, csv_format: bool = None):
+       delimiter_char: str = None, csv_format: bool = None, pipe_format: Format = None):
+    if pipe_format:
```

Comment:
This needs to error in case both `format` and any of the csv format options are given

Reply:
See below. Currently old and new format can be used simultaneously on purposes to avoid breaking changes with custom implementations. It uses the strategy:
- when `pipe_format` is not given, use the old parameters (`header`, `delimiter_char`, `csv_format` etc.)
- when `pipe_format` is given, it overwrites the default parameters when the args are set.

Reply:
This behavior is now changed as suggested.